samples/algorithms/QuantumHelloWorld.qs

```diff
@@ -0,0 +1,27 @@
+/// # Quantum Hello World!
+///
+/// This Q# code generates a random bit by setting a qubit in a superposition of
+/// the computational basis states |0〉 and |1〉, and returning the measurement
+/// result.
```

Comment:
Nit: this comment can be more compact and scannable by moving "result." to the previous line. We don't have to be religious with not exceeding 80 characters per line.

Reply:
Updated.